### PR TITLE
Add dynamodb commands

### PR DESCRIPTION
**Pull request by satyarohith**
This PR implements the following command:
`dynamodb_list` - List the DB tables under your account.

Merge #6 before #7 .

### CODE REVIEW

File: aws/dynamodb/dynamodb_list.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * @return {Promise<SlackBodyType>} Response body
+ */
+async function _command(params, commandText, secrets = {}) {
+  const {awsAccessKey, awsSecretKey, awsRegion} = secrets;
+
+  if (!awsAccessKey || !awsSecretKey || !awsRegion) {
+    return {
+      response_type: 'ephemeral',
+      text:
+        'You must create secrets for `awsAccessKey`, `awsSecretKey` and `awsRegion` to use this command '
```
Comment (niks3089) [inline aws/dynamodb/dynamodb_list.js]: Using `/nc secret`

File: aws/dynamodb/dynamodb_list.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  const ddb = new aws.DynamoDB({
+    apiVersion: '2012-08-10',
+    accessKeyId: awsAccessKey,
+    secretAccessKey: awsSecretKey,
+    region: awsRegion
+  });
+
+  try {
+    const {promisify} = require('util');
+    const listTablesAsync = promisify(ddb.listTables).bind(ddb);
```
Comment (niks3089) [inline aws/dynamodb/dynamodb_list.js:31]: Move this to the top of the file

File: aws/ec2/ec2_list.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    apiVersion: '2016-11-15',
+    accessKeyId: awsAccessKey,
+    secretAccessKey: awsSecretKey,
+    region: awsRegion
+  });
+
+  try {
+    const {promisify} = require('util');
+    const describeInstancesAsync = promisify(ec2.describeInstances).bind(ec2);
+
```
Comment (niks3089) [inline aws/ec2/ec2_list.js]: Nit: Remove unwanted new lines

**Review by niks3089 — CHANGES_REQUESTED** [review]

File: aws/ec2/ec2_list.js (hunk 3 above)
Comment (satyarohith) [reply aws/ec2/ec2_list.js]: Fixed in #6

**Review by niks3089 — APPROVED** [review]

**Pull request merged**